flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/runtime/CodeGenOperatorFactory.java

```diff
@@ -59,7 +59,12 @@ public ChainingStrategy getChainingStrategy() {
 
 	@Override
 	public boolean isOperatorSelectiveReading() {
-		return InputSelectable.class.isAssignableFrom(generatedClass.getClass(Thread.currentThread().getContextClassLoader()));
+		return InputSelectable.class.isAssignableFrom(getStreamOperatorClass(Thread.currentThread().getContextClassLoader()));
+	}
+
+	@Override
+	public Class<? extends StreamOperator> getStreamOperatorClass(ClassLoader classLoader) {
```

Comment:
this method is inconsistent with `isOperatorSelectiveReading()` api that just uses `getContextClassLoader()`. Please make them consistent and use the same mechanism: either passing `classLoader` in both places or using `getContextClassLoader` in both.

Reply:
Because a general method `getStreamOperatorClass()` is added, `isOperatorSelectiveReading()` is not needed and can be dropped. It is planned to drop `isOperatorSelectiveReading()` from the PR of the JIRA (https://issues.apache.org/jira/browse/FLINK-13051), so no changes had been made to make them consistent. But the best thing to do is to keep them consistency first whether `isOperatorSelectiveReading()` is dropped or not. I will solve it.